task4/CarbonIntensityAPI/Request1/Description1.txt

```diff
@@ -0,0 +1 @@
+Выходные данные: данные об интенсивности выбросов углерода в последние полчаса.
```

Comment:
Please avoid storing comments is Russian, In an international project, which is 98 per cent of all projects on the market, any language  but English is discouraged in team communications and tech documentation